internal/flagset/installer.go

```diff
@@ -140,10 +141,9 @@ func ProcessInstallerGenericFlags(cmd *cobra.Command) (InstallerGenericFlags, er
 	log.Println("option.kubefirst.experimental", experimentalMode)
 	flags.ExperimentalMode = experimentalMode
 
-	// TODO: reintroduce the next 3 lines after #511 is closed
-	//if viper.GetBool("github.enabled") && flags.BotPassword == "" {
-	//	return InstallerGenericFlags{}, fmt.Errorf("must provide bot-password argument for github installations of kubefirst")
-	//}
+	if viper.GetBool("github.enabled") && flags.BotPassword == "" {
+		return InstallerGenericFlags{}, fmt.Errorf("must provide bot-password argument for github installations of kubefirst")
```

Comment:
unit test will complain of Errorf, errors.New() will do the trick

Reply:
test just passed: 
![image](https://user-images.githubusercontent.com/53096417/194377076-c77e3dee-8d2b-4ea9-8207-a89bb01ae79e.png)
Do we need to change? Is there any real impact?

Reply:
It probably complains when used Upper case later on the message.

Reply:
no real impact, but its nice to use the correct function for the correct job. 

in short, when there is a "f" in the function name in the `fmt` package, it usually means we can pass arguments to the string like ("%d", myInt).

https://pkg.go.dev/fmt#Errorf

Reply:
feel free to merge with this warning, I can fix it later

Reply:
I added a discussion here if there are mode details on the choice: https://github.com/kubefirst/kubefirst/discussions/533

_I think language is tricky, I don't subscribe on the idea of it been "wrong" to be worth to be fixed. It seems to be at most a change with marginal performance gain._   

Anyway, if these is a new standard to be followed let's try to communicate the value of the change and if it is worth the refactory. Not all rules can be simply generalized, if so we need to context to follow the decision. 

As if you see only this line in the PR, it may be be "simple change", but as the rest of the code base also use the  "old way", so does this means we need to change all code we touch that has the older way? 

In theory, this PR just "uncommented" a part of the code.